lib/turbostreamer/encoders/oj.rb

```diff
@@ -66,12 +66,17 @@ def inject(string)
       # For string containing key and value like `"key":"value"`
       # OJ stream writer does NOT allow writing the whole string
       # But instead requiring key and value to be input separately
-      key_value_match_data = /\A"(\w+?)":(.+)\Z/.match(string)
+      key_value_match_data = /\A"(\w+?)":.+\Z/.match(string)
```

Comment:
Do we even care about anything after the semicolon? The regex doesn't need to scan to the end of the string, we could just do `\A"(\w+?)":/`